projects/client/Unit/RabbitMQ.Client.Unit.csproj

```diff
@@ -0,0 +1,178 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@Scooletz This csproj shouldn't be part of the PR, since the vs-support branch is using project.json/xproj instead.